content/en/Platform Deep Dive/Cobalt Account/account-settings.md

```diff
@@ -66,42 +55,47 @@ To manage two-factor authentication (2FA) settings for your account, select **Se
 
 #### Enable Two-Factor Authentication
 
-Even if your organization hasn't enforced 2FA for all users, we recommend that you enable it to enhance your account security. If you're using [SAML SSO](/getting-started/sign-in/#saml-sso) to sign in, you don't need to turn on 2FA.
+If you're **signing in with your email and password**, you can enable two-factor authentication (2FA) to protect your account. 2FA adds another layer of security to your account by requiring more than just a password to sign in.
 
-![Configure 2FA for your account](/deepdive/SecuritySettings.png "Two-factor authentication page")
+- Even if your organization doesn't [enforce 2FA](/platform-deep-dive/collaboration/organization/organization-settings/enforce-2fa/) for all users, we recommend that you enable it to enhance your account security.
+- If you're signing in through single sign-on, 2FA doesn't apply. This includes [SAML SSO](/getting-started/sign-in/#saml-sso) and Google authentication.
 
 To enable 2FA on your account:
 
 1. Navigate to https://app.cobalt.io/settings/security/2fa.
-    - You should see **Disabled** under **Status**.
-1. Enter your current password, and select **Enable two-factor authentication**.
-    - {{% forgot-password-reset %}}
-1. Scan the QR code using an authenticator app on your mobile device, such as [Authy](https://authy.com/) or [Google Authenticator](https://support.google.com/accounts/answer/1066447).
-1. Enter a 6-character code that your authenticator app generates.
-1. Select **Enable two-factor authentication** to confirm.
+1. Under **Two-Factor Authentication (2FA)**, select **Manage**.
+1. To add a new authenticator device, select **Add New OTP**.
+1. Follow the instructions on the screen to complete the process. Use an authenticator of your choice, such as [Authy](https://authy.com/) or [Google Authenticator](https://support.google.com/accounts/answer/1066447).
+1. Add one or more backup devices to avoid being locked out of your account. Select **Add New OTP** to proceed.
+
+Now, each time you sign in to Cobalt, you must enter a one-time code from your authenticator app. If you have problems signing in with 2FA, see our [troubleshooting tips](/platform-deep-dive/cobalt-account/account-recovery/#problems-with-two-factor-authentication).
+
+#### Manage 2FA Devices
 
-Now, each time you sign in to Cobalt, you need to enter a one-time code from your authenticator app. You can turn on the option to remember your device for 30 days.
+When you add a new 2FA device, it appears under **Current Devices**. Here, you can see devices with an authenticator app that generates one-time codes required for authentication to your Cobalt account.
```

Comment:
we don't have Current Devices right now - that is future. https://www.loom.com/i/aea1deb7297a4a6785fcb428636ece28

Reply:
Removed this for now.